data/uking_functions.csv

```diff
@@ -85850,9 +85850,9 @@
 0x0000007101099918,sub_7101099918,116,_ZThn664_N4ksys3res4DropD0Ev
 0x0000007101099b00,nullsub_4471,4,
 0x0000007101099b04,nullsub_4472,4,
-0x0000007101099b08,Blifecondition::parse,1296,
-0x000000710109a018,Blifecondition::parseArray,448,
-0x000000710109a1d8,ResourceBlifecondition::parse,28,
+0x0000007101099b08,Blifecondition::parse,1296,_ZN4ksys3res13LifeCondition5parseEPhmPN4sead4HeapE
```

Comment:
```suggestion
0x0000007101099b08,Blifecondition::parse,1296,_ZN4ksys3res13LifeCondition6parse_EPhmPN4sead4HeapE?
```

(this should be `parse_` and the ? at the end marks the function as a minor non-matching)